gsa/src/web/components/form/checkbox.js

```diff
@@ -49,19 +49,47 @@ const CheckboxComponent = ({
   toolTipTitle,
   unCheckedValue,
   ...other
-}) => (
-  <StyledElement>
-    <Divider title={toolTipTitle}>
-      <StyledInput {...other} disabled={disabled} type="checkbox" />
-      {isDefined(title) && (
-        <StyledTitle data-testid="checkbox-title" disabled={disabled}>
-          {title}
-        </StyledTitle>
-      )}
-      {children}
-    </Divider>
-  </StyledElement>
-);
+}) => {
+  const [value, setValue] = useState(undefined);
```

Comment:
I guess it should be initialized with the value of a prop named checked (i am not sure if it must be named checked or value). E.g. `<Checkbox checked={true} ... />`